js/libs/core.js

```diff
@@ -161,6 +237,14 @@ Fliplet.FormBuilder = (function() {
         canHide: {
           type: Boolean,
           default: true
+        },
+        source: {
```

Comment:
We should be more specific in what these represent, e.g. `defaultValueSource` and `defaultValueKey`. @tonytlwu do you agree?

Reply:
`defaultValueSource` would be better indeed.